pyproject.toml

```diff
@@ -235,20 +237,23 @@ warn_unused_configs = true
 warn_unused_ignores = true
 
 [[tool.mypy.overrides]]
-disable_error_code = "attr-defined"
-module = ["tests.*.test_sqlalchemy", "tests.*.test_sqlalchemy"]
-
-[[tool.mypy.overrides]]
-disable_error_code = ["arg-type", "misc", "valid-type", "var-annotated"]
-module = "tests.unit.test_contrib.test_sqlalchemy.test_dto"
+module = ["tests.examples.*", "tests.docker_service_fixtures"]
+ignore_errors = true
 
 [[tool.mypy.overrides]]
 disallow_untyped_decorators = false
-module = "tests.*"
+module = ["tests.unit.test_kwargs.test_reserved_kwargs_injection"]
```

Comment:
nit: let's first declare `module` and then all other settings. Right now it is inconsistent.